src/main/webapp/components/DataSharing.js

```diff
@@ -0,0 +1,157 @@
+import { Component } from 'react';
+import { hh, h1, div } from 'react-hyperscript-helpers';
+
+import { WizardStep } from './WizardStep';
+import { InputFieldRadio } from './InputFieldRadio';
+import { InputFieldText } from './InputFieldText';
+import { isEmpty } from '../util/Utils';
+
+export const DataSharing = hh(class DataSharing extends Component {
+
+  state = {};
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      formData: {
+        sharingPlan: '',
+        databaseControlled: '',
+        databaseOpen: ''
+      },
+      errors: {
+        sharingPlan: true
+      }
+    }
+
+  }
+
+  componentDidCatch(error, info) {
+    console.log('----------------------- error ----------------------')
+    console.log(error, info);
```

Comment:
Also ... this seems like a great place for a utility abstraction, something like `spinnerService` but for error handling. It might also be a precursor for generic error handling across the app.

Reply:
I'll create a story for that!